src/Files/Strings/en-US/Resources.resw

```diff
@@ -2744,4 +2744,7 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="CloseOtherTabs" xml:space="preserve">
     <value>Close other tabs</value>
   </data>
-</root>
+  <data name="SidebarDocuments" xml:space="preserve">
```

Comment:
```suggestion
  <data name="Documents" xml:space="preserve">
```
Can you edit the code to look for `Documents`?

Reply:
```cs
switch (libName)
{
	case "Documents":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Pictures":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Music":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	case "Videos":
		tabLocationHeader = $"Sidebar{libName}".GetLocalized(); // Show localized name
		break;
	default:
		tabLocationHeader = library.Text; // Show original name
		break;
}
```
@yaichenbaum This is where it's getting the SidebarDocuments string from. Did you want me to go ahead and change the others as well to keep it consistent?

Reply:
That's a good idea, thank you!